src/Calculator.jsx

```diff
@@ -209,52 +224,52 @@ class Calculator extends Component {
             <Icon onClick={() => this.handleClick('Bottle')}>
               <Img src="/bottle.png" alt="bottle" />
               <Pic1 show={this.state.show}>
-                <Value>x {this.state.Bottle}</Value>
+                <Value>x {this.state.userChoices['Bottle']}</Value>
               </Pic1>
             </Icon>
             <Icon onClick={() => this.handleClick('Cup')}>
               <Img src="/glass.png" alt="glass" />
               <Pic2 show={this.state.show}>
-                <Value>x {this.state.Cup}</Value>
+                <Value>x {this.state.userChoices['Cup']}</Value>
               </Pic2>
             </Icon>
             <Icon onClick={() => this.handleClick('Toilet')}>
               <Img src="/flush.png" alt="flush" />
               <Pic3 show={this.state.show}>
-                <Value>x {this.state.Toilet}</Value>
+                <Value>x {this.state.userChoices['Toilet']}</Value>
               </Pic3>
             </Icon>
           </IconContainer>
           <IconContainer>
             <Icon onClick={() => this.handleClick('Washing hands')}>
               <Img src="/washing-hands.png" alt="washing hands" />
               <Pic4 show={this.state.show}>
-                <Value>{this.state['Washing hands']} Mins</Value>
+                <Value>{this.state.userChoices['Washing hands']} Mins</Value>
               </Pic4>
             </Icon>
             <Icon onClick={() => this.handleClick('Shower')}>
               <Img src="/shower.png" alt="shower" />
               <Pic5 show={this.state.show}>
-                <Value>{this.state.Shower} Mins</Value>
+                <Value>{this.state.userChoices['Shower']} Mins</Value>
               </Pic5>
             </Icon>
             <Icon onClick={() => this.handleClick('Faucet tap')}>
               <Img src="/plumbing-pipe.png" alt="plumbing pipe" />
               <Pic6 show={this.state.show}>
-                <Value>{this.state['Faucet tap']} Mins</Value>
+                <Value>{this.state.userChoices['Faucet tap']} Mins</Value>
               </Pic6>
             </Icon>
           </IconContainer>
-          <Calculate>CALCULATE</Calculate>
+          <Calculate onClick={() => this.handleSubmit()}>CALCULATE</Calculate>
         </Activitys>
         <ResultsContainer>
           <Results>
             <ResultsTitle>Your cost</ResultsTitle>
-            <Numbers>5</Numbers>
+            <Numbers>{this.state.cost}</Numbers>
```

Comment:
The numbers of shekels should be limited to 3 digits at max

instead of `0.478375`
it should be `0.478`